docs/using_task_cache_collision_lock.rst

```diff
@@ -75,11 +101,12 @@ Settings in this section will prevent running ``run()`` at the same time for eff
         redis_port=6379
         redis_fail_on_collision=true
 
-4. Set retry parameters. Set following parameters to retry when task
-   failed. Values of ``retry_count`` and ``retry_delay``\ can be set to
-   any value depends on your situation.
+4. Set retry parameters
+
+    Set following parameters to retry when task failed.
+    Values of ``retry_count`` and ``retry_delay`` are set in seconds.
```

Comment:
I think `retry_count` is not set in seconds.

I like the listing, but feel free.

```suggestion
    * ``retry_count``: the max number of retries
    * ``retry_delay``: this value is set in seconds
```